website/content/en/v0.6.2/getting-started/_index.md

```diff
@@ -156,7 +156,7 @@ Install the chart passing in the cluster details and the Karpenter role ARN.
 helm upgrade --install --namespace karpenter --create-namespace \
   karpenter karpenter/karpenter \
   --version v0.6.2 \
-  --set serviceAccount.annotations.eks\.amazonaws\.com/role-arn=${KARPENTER_IAM_ROLE_ARN}
+  --set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"=${KARPENTER_IAM_ROLE_ARN} \
```

Comment:
So we generally don't update the versioned doc directly. Let me have someone else confirm

Reply:
@suket22 I see, this where the "Edit this page" link from the docs site redirected me. If I need to change it somewhere else instead, let me know :)

Reply:
I think you'd also want to update the preview docs too - https://github.com/aws/karpenter/tree/main/website/content/en/preview

Reply:
I agree that the changes should go in both places. Going forward, I think we should always fix the latest docs if they are broken and make sure the change goes into preview as well.